app/views/issues/new.html.erb

```diff
@@ -1,11 +1,12 @@
 <div class="page-container">
 <% if @line_type %>
 	<h1>Report <%= @line_type.capitalize %> Issue</h1>
+<% elsif @line_id %>
+	<% @line=Line.find_by_onestop_id(@line_id) %>
```

Comment:
Logic should be kept in the controller as much as possible, especially anything that does a DB query like this. I'd recommend moving it into your controller, and then in the view you can just do `<% elsif @line %>`